### PR TITLE
#1-4Task. anna.balkovskaya

### PR DESCRIPTION
**Pull request by Balkanna**
https://travis-ci.org/github/Balkanna/js-assignments

### CODE REVIEW

File: task/02-numbers-tasks.js [hunk 1]
```diff
@@ -221,7 +226,7 @@ function isPrime(n) {
  *   toNumber(new Number(42), 0) => 42
  */
 function toNumber(value, def) {
-    throw new Error('Not implemented');
+    return Number(value) ? Number(value) : def;
```
Comment (FedorovD) [inline task/02-numbers-tasks.js:229]: not correct, what if the value is '0'

File: task/04-arrays-tasks.js [hunk 2]
```diff
@@ -38,7 +38,9 @@ function findElement(arr, value) {
  *    5 => [ 1, 3, 5, 7, 9 ]
  */
 function generateOdds(len) {
-   throw new Error('Not implemented');
+   const arr = new Array(len);
```
Comment (FedorovD) [inline task/04-arrays-tasks.js:41]: Do not use const in such cases, use let

**Comment by FedorovD** [comment]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by Balkanna** [comment]
Fixed. 
Thank you for the comments!

**Comment by FedorovD** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**